src/screens/SUE/SueMapScreen.tsx

```diff
@@ -135,7 +149,18 @@ export const SueMapScreen = ({ navigation, route }: Props) => {
         clusteringEnabled={true}
         locations={mapMarkers}
         mapStyle={styles.map}
+        isMyLocationButtonVisible={!!locationService}
         minZoom={geoMap?.minZoom}
+        onMyLocationButtonPress={() => {
+          const location = position || lastKnownPosition;
+
+          setUpdatedRegion(true);
+          setCurrentPosition(location?.coords);
+
+          setTimeout(() => {
+            setUpdatedRegion(false);
+          }, 300);
```

Comment:
why is a timeout needed? and why 300 and not 100 like a few lines beneath?

Reply:
Because once the user has displayed his own location on the map, the location on the map is not updated the second time the button is pressed. Therefore, the value of the `updateRegion` state had to change.

There's no particular reason for 300. I updated it to 100.